### DISCUSSION TITLE
Move wiki repository tab to labwc.github.io

### DISCUSSION THREAD

**Discussion by ludgie** [labwc/labwc#1439] [Ideas]
Currently I see that under the wiki tab on this repository there's only the list of distributions that have labwc packaged.

I suggest moving that list to labwc.github.io (maybe adding a new section to the getting-started page or a new page like "how to install") so users when accessing the website can know how to install labwc from their distro

Also, maybe just leave on the wiki tab a note like: "See labwc.github.io" or even disable the wiki tab under repository settings maybe (though unnecessary)

**Comment by ahesford** [comment]
What's wrong with the wiki? It seems like a perfect place to leave a dynamic list of distributions and allow users of those distributions to refine the instructions. Moving the instructions to the main site not only increases maintenance burden as the team will have to review and approve the changes, but the review is likely to be less meaningful because we can't be intimately familiar with every distribution.

**Reply by redtide** [reply]
> Moving the instructions to the main site not only increases maintenance burden

(see also [my previous comment](https://github.com/labwc/labwc/issues/1436#issuecomment-1893499684).)

> I was meaning as website the labwc website, https://labwc.github.io/. The source code is at https://github.com/labwc/labwc.github.io

I know the website repo, I contributed to it once.

**Reply by ludgie** [reply]
Got it, thank you
